docs-v2/mint.json

```diff
@@ -522,7 +522,9 @@
                                 "reference/api/connection/get",
                                 "reference/api/connection/post",
                                 "reference/api/connection/set-metadata",
+                                "reference/api/connection/bulk-set-metadata",
```

Comment:
Do we need new endpoints for this? Can we just support both a single value and an array in the existing routes?

Reply:
That what I would have liked to do but we can't without deprecating this current method. This is because the `connectionId` is passed as part of the URL: 

```
curl --location --request PATCH 'http://localhost:3003/connection/to-override/metadata?provider_config_key=google-calendar' \
--header 'Content-Type: application/json' \
--data '{
    "myMetadata": "bar"
}'
```

Reply:
that's logical. It is mirroring the endpoints to modify metadata for a single connection

```
// single connection
POST /connection/:connectionId/metadata
PATCH /connection/:connectionId/metadata

// multiple connection
POST /connection/metadata
PATCH /connection/metadata
```